components/Layout/Footer.tsx

```diff
@@ -63,39 +63,64 @@ export const Footer = () => (
         </ul>
       </div>
     </div>
-    <div className="my-8">
-      <div className="sm:container mx-auto flex justify-between items-center">
-        <nav className="">
-          <ul className="flex flex-col md:flex-row items-start md:items-center list-disc space-x-6">
-            <li className="list-none">
-              <a href="#">Social media</a>
-            </li>
+    <div className="flex justify-between px-4 py-8 sm:container mx-auto">
+      <ul className="flex flex-col md:flex-row md:space-x-6 list-disc px-4">
+        <li>
+          <a href="#">Social media</a>
+        </li>
 
-            <li>
-              <a href="#">Mobile applications</a>
-            </li>
+        <li>
+          <a href="#">Mobile applications</a>
+        </li>
 
-            <li>
-              <a href="#">About Canada.ca</a>
-            </li>
+        <li>
+          <a href="#">About Canada.ca</a>
+        </li>
 
-            <li>
-              <a href="#">Terms and conditions</a>
-            </li>
+        <li>
+          <a href="#">Terms and conditions</a>
+        </li>
 
-            <li>
-              <a href="#">Privacy</a>
-            </li>
-          </ul>
-        </nav>
-        <div>
-          <Image
-            width="200px"
-            height="40px"
-            src="/wmms-blk.svg"
-            alt="Symbol of the Government of Canada"
-          />
-        </div>
+        <li>
+          <a href="#">Privacy</a>
+        </li>
+      </ul>
+      <div className="hidden md:block">
+        <Image
+          width="200px"
+          height="40px"
+          src="/wmms-blk.svg"
+          alt="Symbol of the Government of Canada"
+        />
+      </div>
+    </div>
+    <div className="flex justify-between">
+      <div className="flex items-center md:hidden py-8 md:pt-0">
+        <p className="px-4">Top of page</p>
```

Comment:
This says "top of page" but I don't see that anywhere in the page, it this text supposed to be visible somewhere?

Reply:
Visible on mobile only, as in the GC design mockups. That's what the `md:hidden` is doing